Description: fix leak of task_in_scsi flag 
Author: Chuck Short <zulcss@ubuntu.com>
Fowarded: Not Needed
diff -Naurp tgt-1.0.43.orig/usr/target.c tgt-1.0.43/usr/target.c
--- tgt-1.0.43.orig/usr/target.c	2014-01-07 07:16:21.000000000 -0500
+++ tgt-1.0.43/usr/target.c	2014-04-09 19:25:53.569907620 -0400
@@ -1454,7 +1454,7 @@ enum mgmt_req_result target_mgmt_request
 	}
 
 	if (err)
-		return err;
+		return MGMT_REQ_FAILED;
 	else if (send)
 		return MGMT_REQ_DONE;
 
